### PR TITLE
Fix name of TUI experimental ENV variable

### PR DESCRIPTION
**Pull request by gmile**
As PR title suggests, it's a small fix to the name of ENV variable.

### CODE REVIEW

**Review by wingyplus — APPROVED** [review]

**Comment by wingyplus** [comment]
@gmile Thank you for your contribution! Current we cannot merge this PR because the commit did not get signed. Please checkout https://github.com/dagger/dagger/blob/main/CONTRIBUTING.md for contribution guideline. After it pass, we can get it merge. ^^

**Comment by gmile** [comment]
Thanks for the heads up, @wingyplus! The DCO issue has now beed addressed 🙇

**Review by gerhard — APPROVED** [review]
Thank you @gmile 💪

**Pull request merged**
